packages/shared/lib/db/database.ts

```diff
@@ -2,32 +2,7 @@ import knex from 'knex';
 import type { Knex } from 'knex';
 import { metrics, retry } from '@nangohq/utils';
 import type { Pool } from 'tarn';
-
```

Comment:
This was moved out because currently trying to create a migration file fails using the cli 

```
Invalid or unexpected token
/Users/khaliqgant/Projects/nango/short-term-feature/packages/utils/dist/environment/detection.js:7
export const isCloud = process.env['NANGO_CLOUD']?.toLowerCase() === 'true';
                                                  ^

SyntaxError: Invalid or unexpected token
    at require.extensions..js (/Users/khaliqgant/Projects/nango/short-term-feature/node_modules/knex/bin/cli.js:465:18)
```

due to the usage of `utils` in the database file which gets called by the knexfile.cjs file